server/validators/goalTargetCompletionDateValidator.ts

```diff
@@ -1,4 +1,4 @@
-import moment from 'moment'
+import { isBefore, isValid, parse } from 'date-fns'
```

Comment:
I took this opportunity to replace `moment` with `date-fns` seeing I was in this file anyway 👍